.github/workflows/build_release.yml

```
@@ -0,0 +1,173 @@
+              <sup>Contributed by @bsojka</sup>
+              ___
+              #### SHA256 checksum of `${{ env.PKG_DOT_NAME }}.pkg`:
+              ```
+              ${{ env.CHECKSUM }}
+              ```
+          files: |
+            ${{ github.workspace }}/artifacts/*.pkg
+            ${{ github.workspace }}/artifacts/*.pkg.sha256
+
+      - name: Upload package
+        uses: actions/upload-artifact@v4 # v4
```

yamllint

[warning] too few spaces before comment

(comments)